adb_graphics/specs.py

```diff
@@ -298,6 +298,13 @@ def rainbow12_colors(self) -> np.ndarray:
                           ([15, 18, 20, 25, 50, 60, 70, 80, 85, 90, 100, 120])
         return np.concatenate((grays, ncar))
 
+    @property
+    def rainbow12_reverse(self) -> np.ndarray:
+
+        ''' Default color map for min helicity '''
+
+        return np.flip(self.rainbow12_colors, 0)
```

Comment:
Sorry I didn't get back to you on this question in a timely manner. You found it! Thanks!